erdjs/src/dapp/walletConnectProvider.ts

```diff
@@ -138,6 +145,7 @@ export class WalletConnectProvider extends EventTarget implements IDappProvider
 
     private async signTransaction(method: any, params: any): Promise<any> {
         if (!this.walletConnector) {
+            Logger.error("signTransaction: Wallet Connect not initialised, call init() first");
```

Comment:
Optionally, these 3 lines can be extracted to `this.guardInitialized()` (a bit of duplication):

```
guardInitialized() {
    if (!this.walletConnector) {
            Logger.error("sendTransaction: Wallet Connect not initialised, call init() first");
            throw new Error("Wallet Connect not initialised, call init() first");
    }
}
```